File: C++/DSA/LinearSearch.cpp
#include<iostream>

using namespace std;

int main(){
    int n,i,a[100],search,flag;
    cout<<"\n Enter The Range Of The Array- ";
    cin>>n;
    for(i=0;i<n;i++)
    {
        cout<<"\n Enter Position "<<i<<" Element- ";
        cin>>a[i];
    }
    cout<<"\n The Array is- ";
    for(i=0;i<n;i++)
    {
        cout<<a[i]<<" ";
    }
    i=0;
    cout<<"\n Enter The Element you want to search? \n";
    cin>>search;
    while(i<n)
    {
        if(a[i]==search)
        {
            flag=1;
        }
        i++;
    }
    if(flag==1)
    {
        cout<<"\n The Element is found.";
    }
    else{
        cout<<"\n The Element is not found.";
    }
}

File: Java/DSA/LinearSearch.java
package Java.DSA;

import java.util.*;

public class LinearSearch {
    public static void main(CharecterType args[]) {
        int a[] = new int[100], i, n, b;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter The Number of Value in Array-");
        n = sc.nextInt();
        for (i = 1; i <= n; i++) {
            System.out.println("Enter The " + i + " Value- ");
            a[i] = sc.nextInt();
        }
        System.out.println("What Value you want to search in the Array?");
        b = sc.nextInt();
        i = 1;
        while (i <= n) {
            if (a[i] == b) {
                System.out.println("The Position of the element " + b + " is " + i);
                break;
            }
            i++;
        }
    }
}
